### DISCUSSION TITLE
Azure DevOps Project Configuration for Enterprise Scale

### DISCUSSION THREAD

**Discussion by pwalleni** [Azure/Enterprise-Scale#897] [Q&A] [answered]
Hi! How do you recommend we configure an Azure DevOps project for managing the people, work items, and code to support an Enterprise Scale implementation over time?

**Comment by jtracey93** [comment, marked as answer]
Hey @pwalleni,

We do have some guidance here in CAF https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/landing-zone/design-area/platform-automation-devops

But this is also an area we are looking to expand upon in the first half of this year also 👍

I will mention this to the team as evidence and may get them to reach out when we have something drafted for you to review.

If you were to give us a top 5/10 things to cover, what would they be?

Thanks

Jack

**Reply by pwalleni** [reply]
Thanks for the link. The tricky part is usually team topologies and responsibility assignments. How do we divide the work between teams that are collectively responsible for continuous deployment of Enterprise Scale? How many people are required to support a deployment scaled up to thousands of subscriptions/workloads? What are typical work items in the backlog of those teams?

Also, do you propose we manage the work with GitHub Projects or Azure DevOps Projects?

**Reply by jtracey93** [reply]
This guidance is also worth a read, whilst not specific to ESLZ today. It provides a good overview of common models and approaches. https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/operating-model/

**Reply by jtracey93** [reply]
Also https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/organize/organization-structures

**Reply by pwalleni** [reply]
Thanks!

**Reply by MartinDBakerCDW** [reply]
In terms of managing code - deployment/mechanisms/approval processes etc, have a look here for an excellent approach: https://jacktracey.co.uk/terraform-with-azure-devops/
